test/library/packages/LinearAlgebra/performance/lu/lu-perf.perfexecopts

```diff
@@ -0,0 +1,2 @@
+--m=1000 --iters=100 -nl 1 #m1000
+--m=10000 --iters=10 -nl 1 #m10000

```

Comment:
Let's remove the larger size, and rename the file to have some "lu" in it somewhere. Current transpose tests also create files named "mXXX", and this one conflicts with those. (I think transpose's filenames should also change but not in this PR)